src/styles/theme/oc-autocomplete.scss

```diff
@@ -48,5 +48,11 @@
       padding: $global-small-gutter;
       background-color: $global-muted-background;
     }
+
+    &:hover .uk-text-meta,
```

Comment:
since we are applying our own logic/styling to the class uk-text-meta: would it make sense to create our own class oc-text-meta?

Reply:
Not really sure if it's needed in this case when we apply only one style. In case we would add more then I'd definitely agree. Or am I just missing some other place were we define other styles for this class?

Reply:
You can drop the whole SCSS if you follow what I mentioned above. Generally, `uk-light` and `uk-dark` (https://getuikit.com/docs/inverse) is the right approach for switching colors.

Reply:
`uk-light` doesn't overwrite colour from `uk-text-meta`